staging/src/k8s.io/kubectl/pkg/cmd/util/printing.go

```diff
@@ -22,6 +22,11 @@ import (
 	"k8s.io/kubectl/pkg/util/templates"
 )
 
+const (
+	YellowColor = "\u001b[33;1m"
+	ResetColor  = "\u001b[0m"
+)
```

Comment:
1. How come this is new - we already use yellow warnings elsewhere, right?
1. Do we detect whether the output is to a terminal / what capabilities that terminal has?
    If writing to a pipe or file, we shouldn't embed ANSI-style colors. If writing to a nonstandard terminal - probably also not.